package.json

```diff
@@ -30,7 +30,8 @@
     "test:update": "npm test -- --updateSnapshot --coverage",
     "validate": "kcd-scripts validate",
     "setup": "npm install && npm run validate -s",
-    "dtslint": "dtslint typings"
+    "dtslint": "dtslint typings",
+    "precommit": "kcd-scripts pre-commit"
```

Comment:
Could you revert these changes?

Reply:
Humpf sorry. I was trying to see what was wrong with my commit on typescript definitions...